### PR TITLE
Feature/call to action

### PR DESCRIPTION
**Pull request by DivyanshiSingh**
Fixes: #102 

- Duplicated the content from the Quickstart page to the call to action on the landing page.
- Added a button to navigate to /quickstart.
- Added test cases for the call to action component.

@mairin, please review

### CODE REVIEW

File: src/components/Welcome/components/WelcomeDevelopers/WelcomeDevelopers.js [hunk 1]
```diff
@@ -1,10 +1,32 @@
...
+        <p>
+          ChRIS is an <strong>open source platform</strong> for medical
+          analytics in the cloud, democratizing the development of image
+          processing apps within an ecosystem following
+          <strong> common standards, rather than disparate silos</strong>.
+        </p>
+        <p>
+          A ChRIS Developer account enables you to submit your image processing
+          application as a containerized ChRIS plugin and share it with the
+          broader ChRIS community of researchers and clinicians.{" "}
```
Comment (jennydaman) [inline src/components/Welcome/components/WelcomeDevelopers/WelcomeDevelopers.js:21]: Omg thank you for showing me how to do `{" "}`

**Comment by jennydaman** [comment]
@mairin In my opinion, the wording here is too grandiose. Do you think we should make it more concise or keep it as-is?

> ChRIS is an <strong>open source platform</strong> for medical analytics in the cloud, democratizing the development of image      processing apps within an ecosystem following **common standards, rather than disparate silos**.
>
> A ChRIS Developer account enables you to submit your image processing application as a containerized ChRIS plugin and share it with the broader ChRIS community of researchers and clinicians.

Suggestion:

> _ChRIS_ is an **open-source platform** for medical analytics in the cloud, democratizing the development and usage of image processing software within an ecosystem following **common standards.**
>
> A _ChRIS_ developer account enables you to share your analysis workflows as containerized software with the _ChRIS_ community of researchers and clinicians.

**Comment by mairin** [comment]
@jennydaman Agreed, and the text changes look perfect to me!

**Comment by DivyanshiSingh** [comment]
Ok, I will replace the pre-existing content with the content given in the suggestion with the same formatting.

**Comment by DivyanshiSingh** [comment]
I have made the changes.

**Review by jennydaman — CHANGES_REQUESTED** [review]
Full-column width at 1920p screen horizontally is too wide. I think you should add empty white-space (or better, a graphical element) to the left or right of the text. You could simply copy/paste the code from "Focus on your data. Not the tools." section.

Full-page screenshot

**Comment by DivyanshiSingh** [comment]
I have added white spaces @jennydaman

File: src/components/Welcome/components/WelcomeDevelopers/WelcomeDevelopers.css [hunk 2]
```diff
@@ -1,14 +1,33 @@
 .welcome-developers {
   /* display */
+  display:flex;
+  flex-direction:column;
+  width:70% !important;
```
Comment (jennydaman) [inline src/components/Welcome/components/WelcomeDevelopers/WelcomeDevelopers.css:5]: `!important` doesn't seem necessary here. Are you using `!important` to match the style of the rest of the CSS code in this repo?

File: src/components/Welcome/components/WelcomeDevelopers/WelcomeDevelopers.js [hunk 3]
```diff
@@ -1,10 +1,31 @@
...
+          image processing software within an ecosystem following
+          <strong> common standards.</strong>.
+        </p>
+        <p>
+          A <i>ChRIS</i> developer account enables you to share your analysis
+          workflows as containerized software with the <i>ChRIS</i> community of
+          researchers and clinicians. <strong>Join us!</strong>
+        </p>
+      </div>
+      <Link to="/quickstart" className="btn callToAction-btn btn-primary">
```
Comment (jennydaman) [inline src/components/Welcome/components/WelcomeDevelopers/WelcomeDevelopers.js:24]: There's already a "Learn More" button nearby but it doesn't work.

We can figure out a good layout in a future PR, what you have here is good.

screenshot

File: src/components/Welcome/components/WelcomeDevelopers/WelcomeDevelopers.css (hunk 2 above)
Comment (DivyanshiSingh) [reply src/components/Welcome/components/WelcomeDevelopers/WelcomeDevelopers.css:5]: I will remove It as its not required. I think the space is fine right?

Comment (jennydaman) [reply src/components/Welcome/components/WelcomeDevelopers/WelcomeDevelopers.css:5]: Looks fine to me

Comment (DivyanshiSingh) [reply src/components/Welcome/components/WelcomeDevelopers/WelcomeDevelopers.css]: I have removed  !important.

File: src/components/Welcome/components/WelcomeDevelopers/WelcomeDevelopers.js (hunk 3 above)
Comment (mairin) [reply src/components/Welcome/components/WelcomeDevelopers/WelcomeDevelopers.js:24]: I think the "Learn more" in the "expand the reach..." area should be a "sign up" button not a learn more. the CTA there is to create a ChRIS store account. The Learn more above should point to chrisproject.org

**Comment by mairin** [comment]
I can't easily build this locally to take a look right now, but based on the screenshots the only comment I have is to make sure the margins of the "expand the reach of your image processing software" are the same as they are for the "focus on your data, not the tools" section. 

This doesn't appear to be an issue in the screenshots but chrisstore.co isn't loading the fonts correctly so when this gets deployed I'd double check and make sure the right font is getting loaded (should be open sans and overpass.) 

@DivyanshiSingh are you planning a separate PR for the icon fixes?

**Comment by DivyanshiSingh** [comment]
@mairin the margin is the same, but you can recheck. And yes I have created a separate PR #124 for the icon and plugin changes. Do you want me to merge them in one?

**Comment by mairin** [comment]
> @mairin the margin is the same, but you can recheck. And yes I have created a separate PR #124 for the icon and plugin changes. Do you want me to merge them in one?

Nope, I think its fine to keep them separate, just didn't know your plan there. Looks good to me now!

**Review by mairin — APPROVED** [review]
Looks good!

**Review by jennydaman — APPROVED** [review]

**Pull request merged**